test/test_tensorboard.py

```diff
@@ -0,0 +1,454 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from common_utils import run_tests
+
+from torch.utils.tensorboard import x2num, SummaryWriter
+import torch
+import numpy as np
+import unittest
+from torch.utils.tensorboard import summary
+from torch.utils.tensorboard.utils import _prepare_video, convert_to_HWC
+import pytest
+# from torch.utils.tensorboard import SummaryWriter
+# from .expect_reader import compare_proto
+import os
+import sys
+import matplotlib.pyplot as plt
+# from torch.utils.tensorboard import x2num
+
+
+class PyTorchNumpyTest(unittest.TestCase):
+    def test_pytorch_np(self):
+        tensors = [torch.rand(3, 10, 10), torch.rand(1), torch.rand(1, 2, 3, 4, 5)]
+        for tensor in tensors:
+            # regular tensor
+            assert isinstance(x2num.make_np(tensor), np.ndarray)
```

Comment:
In tests, you should use `self.assertIsInstance` instead of `assert`, you will get more useful diagnostics if it fails

Reply:
Replaced 35 occurrences into unittest.Testcase based assertion.